styles.css

```diff
@@ -825,3 +825,151 @@ button:active {
   outline: none;
   border: none;
 }
+
+.form-start {
+  width: 375px;
+  height: 930px;
+}
+
+.form-header {
+  width: 327px;
+  height: 21px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 24px;
+  margin-top: 120px;
+}
+
+#full-name {
+  width: 327px;
+  height: 32px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #979493;
+  position: absolute;
+  margin-top: 270px;
+  margin-left: 24px;
+  display: flex;
+  align-items: center;
+  padding-left: 10px;
+  border-bottom: 1px solid #d8d8d7;
+  border-left: 0;
+  border-top: 0;
+  border-right: 0;
+}
+
+#email {
+  width: 327px;
+  height: 32px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #979493;
+  position: absolute;
+  margin-top: 353px;
+  margin-left: 24px;
+  display: flex;
+  align-items: center;
+  padding-left: 10px;
+  border-bottom: 1px solid #d8d8d7;
+  border-left: 0;
+  border-top: 0;
+  border-right: 0;
+}
+
+textarea {
+  width: 327px;
+  height: 114px;
+  position: absolute;
+  background-color: #fbfbf7;
+  border: 0;
+  padding: 12px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #3c3a39;
+  margin-left: 24px;
+  margin-top: 467px;
+}
+
+.submit {
+  width: 166px;
+  height: 48px;
+  margin-top: 683px;
+  margin-left: 112px;
+  background-color: #ff6b00;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  text-align: center;
+  padding: 12px;
+  font-size: 24px;
+  outline: none;
+  border: none;
+  position: absolute;
+}
```

Comment:
- [ ] kindly consider adding Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template. this will make your work more professional cause you will be meeting the project requirements
kindly choose any to go with from the below 👇 
![hh](https://user-images.githubusercontent.com/60146030/221374786-e79620fb-3b4c-4b6a-a027-08ff79c4c665.PNG)